micrometer/tasks/task-demo-metrics-influxdb/README.md

```diff
@@ -0,0 +1,131 @@
+
+# How to monitor Spring Cloud Tasks
+
+The `task-demo-metrics-influx` project creates a sample Spring Cloud Task (and Spring Batch) enabled for Spring Cloud Data Flow monitoring.  
```

Comment:
> The `task-demo-metrics-influx` project creates a sample Spring Cloud Task (and Spring Batch) with the monitoring infrastructure, so it can be monitored in Spring Cloud Data Flow.